core/src/main/java/io/cucumber/core/cli/CLIOptions.java

```diff
@@ -0,0 +1,58 @@
+package io.cucumber.core.cli;
+
+import org.apiguardian.api.API;
+
+@API(status = API.Status.STABLE)
+public final class CLIOptions {
```

Comment:
CommandLineOptions was a good name. Why did you change it?

Reply:
And don't forget to add some java doc. Gotta explain to people what this is for. And what to use it in combination with.

Reply:
I renamed because I didn't use static import at first, making the code a little bit clumsy. Since I'd used static import, I reverted it to the old name. And included some javadoc too. 😄